### ISSUE TITLE
Debian 9 compatibility

### ISSUE THREAD

**Issue by 1ccs-todd** [pavelmc/sof1u#3]
original source has been updated to Debian 9.  This may be needed for those still wanting to run ScrollOut F1 from Ubuntu.  For Debian 9.x (Stretch) simply use installer iso from www.scrolloutf1.com.

If you have an old Scrollout running from Debian 8, updating Scrollout now will break compatibility.  An update to the Debian base from 8.x to 9.x is required.  Then (re)apply the Scrollout F1 update and restart.  Everything works again!

**Comment by ghost** [comment]
Any other Update?

Great Job, please added skype: luisgarcia562

**Comment by 1ccs-todd** [comment]
The main source at scrolloutf1.com was updated for Debian 9.x+ compatibility. 

If you are running Ubuntu and the main source fails, this subversion will work.

**Comment by ghost** [comment]
Yes, it's with debian now. Thank you

**Comment by ghost** [comment]
Pavel:you recommend using scrolloutF1 for 1600 mailboxes?

thanks

**Comment by stdevPavelmc** [comment]
Hi @jason25, keep in mind that this soft is for a email gateway aka: the mailboxes are not in it, it's just a gateway between the internet and your final internal server. An yes, I have it on production on a client with about +2K accounts in it's main server.

It has a weird & un-triaged quirk on Ubuntu 18.04 over virtualized environments (CT on Proxmox) that it keeps eating RAM until some point in which it lock and needs a reboot. A daily reboot on cron fixes this...

I have very little time to support this (fork) now, so I recommend to go to the updated and original Scroll Out F1 on Debian as support from me is not a safe bet right now.

Cheers, and yes @stdevPavelmc = @pavelmc ;-)

**Comment by stdevPavelmc** [comment]
In the spirit of FLOSS:

If some one want to keep supporting/developing this fork, he or she is free to fork this and I will be happy to place a note about your efforts in My README.md front page.

73

**Comment by ghost** [comment]
yes, I have seen this, I understand that it works like a mail gateway, I have it protecting zimbra, I like it a lot.

but I have seen that it hangs sometimes and I only have it with few accounts.

configure it in a VPS with 8GB RAM with Debian 09

Thank you for your answers.
